### PR TITLE
feat: added grid component

### PR DESCRIPTION
**Pull request by ShekharDhangar**
added gird component

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@import url('/utils.css');
+.grid-container{
+    display: grid;
+    padding: 1rem;
+    border: 1px solid black;
+    grid-template-columns: repeat(2,1fr);
```
Comment (partha8) [inline components/grid/grid.css]: you can extract this (line 6) and put it in box-1, like you did for box-2.

**Review by partha8 — APPROVED** [review]
Everything looks good, but i feel like adding different colors to every item/content was a bit overkill.

File: components/grid/grid.css (hunk 1 above)
Comment (ShekharDhangar) [reply components/grid/grid.css]: made the changes

Comment (ShekharDhangar) [reply components/grid/grid.css]: the different color for grid was just to differentiate

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+@import url('/utils.css');
+.grid-container{
+    display: grid;
+    padding: 1rem;
+    border: 1px solid black;
```
Comment (zoef-yt) [inline components/grid/grid.css:5]: use var for color

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+.item-2{
+    background-color: rgb(27, 169, 226); 
+}
+.item-3{
+    background-color: rgb(15, 160, 104);
+}
+.item-4{
+    background-color: rgba(22, 65, 207, 0.658);
+}
```
Comment (zoef-yt) [inline components/grid/grid.css]: use var for colour

File: components/grid/grid.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  .content-1 {
+    background-color: var(--clr-teal);
+  }
+  
+  .content-2 {
+    background-color: var(--success-clr);
+  }
+  
+  .content-3 {
+    background-color: #1EAAFC;
```
Comment (zoef-yt) [inline components/grid/grid.css]: use var for color

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: components/grid/grid.css (hunk 4 above)
Comment (ShekharDhangar) [reply components/grid/grid.css]: Made the changes. please check

**Review by zoef-yt — APPROVED** [review]
Code is good to merge.

**Pull request merged**